### PR TITLE
Add thread switch trace

### PR DESCRIPTION
**Pull request by jsvapiav**
jira: HYPR-432
Signed-off-by: Hannu Lyytinen <hannux@ssrc.tii.ae>

### CODE REVIEW

**Comment by hlyytine** [comment]
@elmankku can you have a look at this.

File: scripts/build_sel4.sh [hunk 1]
```diff
@@ -31,7 +31,15 @@ ln -rs "${SRCDIR}/easy-settings.cmake" "${BUILDDIR}"
...
 
 # shellcheck disable=SC2068
-./init-build.sh -B . -DAARCH64=1 -DPLATFORM="${PLATFORM}" -DCROSS_COMPILER_PREFIX="${CROSS_COMPILE}" $@
+./init-build.sh \
+  -B . \
+  -DAARCH64=1 \
+  -DPLATFORM="${PLATFORM}" \
+  -DCROSS_COMPILER_PREFIX="${CROSS_COMPILE}" \
+  -DKernelArmExportPMUUser=ON \
+  -DKernelBenchmarks=track_kernel_entries \
```
Comment (elmankku) [inline scripts/build_sel4.sh]: Do we always want to have these options set, or should we have e.g. another make target build with tracing enabled...?

Tracing definitely affect the performance.

Comment (hlyytine) [reply scripts/build_sel4.sh]: Hmmm yeah, no, we don't need them always set. Trying to ponder how to fix it properly...

**Comment by hlyytine** [comment]
How about this?

**Comment by hlyytine** [comment]
How about this?

**Comment by elmankku** [comment]
It's good now

**Pull request merged**